src/main/scala/uk/gov/hmrc/releaser/ArgParser.scala

```diff
@@ -25,7 +25,8 @@ object ArgParser{
                      githubNameOverride:Option[String] = None,
                      tag:Boolean = true,
                      verbose:Boolean = false,
-                     dryRun:Boolean = false)
+                     dryRun:Boolean = false,
+                     releaseNotes: String = "")
```

Comment:
Can we not make this an `Option[String]` and have it default to `None`

Reply:
Yes, we can!